### PR TITLE
Medbay Lobby: mrhugo13 realizes that he is dumb

### PR DESCRIPTION
**Pull request by mrhugo13**
## About The Pull Request
This
dreammaker_2020-02-10_16-00-08
To this
dreammaker_2020-02-10_16-20-24

## Why It's Good For The Game
unconnected scrubber networks is bad and generally cleaning up medbay lobby again

## Changelog
:cl:Vondiech
fix: The scrubber network in medbay should now be working again.
tweak: Cleaned up a few issues and mistakes that i overlooked.
/:cl:

### CODE REVIEW

**Comment by mrhugo13** [comment]
fuck i just noticed another few issues with medbay lobby FUCK

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -75920,6 +75918,10 @@
 /obj/structure/disposalpipe/segment,
 /turf/open/floor/plating,
 /area/maintenance/starboard/secondary)
+"vZr" = (
+/obj/machinery/firealarm,
+/turf/closed/wall,
```
Comment (nemvar) [inline _maps/map_files/MetaStation/MetaStation.dmm]: don't place alarms inside of the wall.

Comment (mrhugo13) [reply _maps/map_files/MetaStation/MetaStation.dmm]: Hopefully fixed now, scream at me again if i did not.

**Comment by Tlaltecuhtli** [comment]
this room is super dumb anyway why does it exist?

**Comment by ATH1909** [comment]
By the way, while we're here, when are you adding back the roundstart suit sensor monitoring console?

**Comment by mrhugo13** [comment]
> 
> 
> By the way, while we're here, when are you adding back the roundstart suit sensor monitoring console?

dreamseeker_2020-02-10_18-49-34

**Comment by cacogen** [comment]
Did you fix the duplicate camera tags between medbay lobby and outside the CMO's office beneath cryo? You should turn on debug verbs and use the verbs under the Mapping tab to see if there's anything you've missed if you haven't already.

**Comment by mrhugo13** [comment]
> 
> 
> Did you fix the duplicate camera tags between medbay lobby and outside the CMO's office beneath cryo? You should turn on debug verbs and use the verbs under the Mapping tab to see if there's anything you've missed if you haven't already.

I swear to god cam tags should be dynamically generated depending on the room they are in instead of depending on some mapper to create another instance of a camera just so they can get the proper camera tag for a room, but either way with what i just pushed should fix the medical lobby c_tag to instead of being "Medbay Central Hallway" to "Medbay Reception".

**Comment by kriskog** [comment]
Merge conflict

**Comment by mrhugo13** [comment]
gimme a moment while i fix this merge conflict

**Comment by mrhugo13** [comment]
fuck this i'll open a new PR once master gets merged again

**Comment by mrhugo13** [comment]
fuck it i'm just gonna make a new PR

**Comment by mrhugo13** [comment]
unless my clean master is lying to me then someone has merged a fix for this so uhhhhhhhhhhhh.

**Comment by cacogen** [comment]
outplayed son

**Pull request closed without merging**